### ISSUE TITLE
Page content scrolls into navbar

### ISSUE THREAD

**Issue by rootulp** [opynfinance/squeeth-monorepo#205]
**Describe the bug**
Page content scrolls into navbar

**To Reproduce**

1. Go to https://squeeth.opyn.co/
1. Scroll down

**Expected behavior**
A. Navbar and page content both scroll down or
B. If page has a static navbar, content scrolls behind navbar

**Screenshots**
<img width="707" alt="Screen Shot 2022-03-01 at 9 43 33 PM" src="https://user-images.githubusercontent.com/3699047/156284946-76c2a1d6-e928-4c6f-824e-1eec6b6a704d.png">

**Additional context**
It may be worth exploring `display: fixed` [here](https://github.com/opynfinance/squeeth-monorepo/blob/main/packages/frontend/src/components/Nav.tsx#L26) or consider using [`<AppBar>`](https://mui.com/components/app-bar/)

**Comment by KMKoushik** [comment]
@rootulp Thanks for raising this issue. It works fine for the chrome browser. I think the issue is with the [backdrop filter](https://github.com/opynfinance/squeeth-monorepo/blob/main/packages/frontend/src/components/Nav.tsx#L31). Doc mentions that backdrop filter is not available for [Mozilla browser](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility).

Will fix it. Appreciate any suggestions!

<img width="1440" alt="Screenshot 2022-03-02 at 3 30 36 PM" src="https://user-images.githubusercontent.com/24666922/156295277-50a5f0d2-812c-4e91-b300-639a2f09976d.png">

**Comment by rootulp** [comment]
You're right! I can reproduce this on Firefox but can't reproduce on Chrome / Brave.
